roles/openshift_router/meta/main.yml

```diff
@@ -12,4 +12,4 @@ galaxy_info:
   categories:
   - cloud
   dependencies:
-  - openshift_facts
+  - openshift_common
```

Comment:
hmm, I'm wondering if the common role is enough here to make this fully standalone...  But at the same time, I wouldn't want to pull in the openshift_master playbook here either.